tests/read_ntfs.rs

```diff
@@ -391,3 +392,19 @@ fn sanitize_grid_with_line_external_code() {
     assert_eq!(1, collections.grid_calendars.len());
     assert_eq!(1, collections.grid_rel_calendar_line.len());
 }
+
+#[test]
+fn ntfs_with_duplicated_ids() {
+    testing_logger::setup();
+    let model =
+        transit_model::ntfs::read("tests/fixtures/ntfs_complete_with_duplicated_ids").unwrap();
+    assert_eq!(2, model.lines.len());
+    testing_logger::validate(|captured_logs| {
+        let warn_logs = captured_logs
+            .iter()
+            .filter(|log| log.level == LogLevel::Warn)
+            .collect::<Vec<_>>();
+        assert_eq!(1, warn_logs.len());
+        assert_eq!("identifier ME:line2 already exists", warn_logs[0].body);
+    });
```

Comment:
Since you copied an entire fixture, you might just test for the entire output result too. Because running the entire process only for one collision, you might have been able to mae the test simpler and do a unit test on `make_collection_with_id` and `make_opt_collection_with_id`.

Reply:
Oh no I will pollute all the collections and test everything. I just had time to do this for the lines quickly yesterday.